airflow/providers/google/cloud/example_dags/example_cloud_build.py

```diff
@@ -99,6 +99,12 @@
         task_id="create_build_from_repo_result",
     )
 
+    create_build_from_file = CloudBuildCreateOperator(
```

Comment:
Can you add this example to guide? https://airflow.readthedocs.io/en/latest/howto/operator/gcp/cloud_build.html